www/less/misc.less

```diff
@@ -82,3 +82,26 @@ a{
 * {
     -webkit-touch-callout: none !important;
 }
+
+//updating typography for h2 and h3
+.content h2 {
+    padding: 20px 0 10px;
+    border-bottom: 1px dotted #eee;
+    margin-bottom: 0;
+}
+
+.content h3 {
+    font-family: -apple-system, HelveticaNeue;
+    font-weight: 700;
+    padding: 15px 0 0!important;
+}
+
+.content h3 a.edit_section_button {
+    display: none !important;
+}
+
+h2 a.edit_section_button,
+h3 a.edit_section_button {
```

Comment:
Maybe no need for 'h3 a.edit_section_button' here because those are being hidden on line 100?